### PR TITLE
feat(components/atom/badge): add secundary type

### PR DESCRIPTION
**Pull request by adriadoz**
### Types of changes

-  New feature (non-breaking change which adds functionality)

### Description, Motivation and Context
- Add secondary by default
- Move demo from playground

### Screenshots - Animations
image

### CODE REVIEW

**Review by oriolpuig — APPROVED** [review]

**Review by nachoAdevinta — APPROVED** [review]

**Comment by andresin87** [comment]
plz wait

File: components/atom/badge/demo/index.js [hunk 1]
```diff
@@ -0,0 +1,509 @@
...
+              transparent
+            />
+            <AtomBadge
+              label="primary"
+              type={atomBadgeTypes.PRIMARY}
+              transparent
+            />
+            <AtomBadge
+              label="primary"
+              type={atomBadgeTypes.SECUNDARY}
```
Comment (andresin87) [inline components/atom/badge/demo/index.js]: SEC`O`NDARY

File: components/atom/badge/demo/index.js [hunk 2]
```diff
@@ -0,0 +1,509 @@
...
+  Code,
+  Bold,
+  Paragraph,
+  Article,
+  UnorderedList,
+  ListItem,
+  DevIcon,
+  Grid,
+  Cell
+} = window.DOC
```
Comment (andresin87) [inline components/atom/badge/demo/index.js]: import from '@s-ui/documantation-library'

**Review by andresin87 — CHANGES_REQUESTED** [review]

**Comment by andresin87** [comment]
Secondary token is only used by RE different than accent color. Let's find some other different approach.

**Comment by andresin87** [comment]
- [x] Playground removed
- [x] Demo made in js.
- [x] Demo articles splitted
- [x] Component constants moved to a separatd file.
- [x] Component Code in its own reserved file.
- [x] Package index.js file reserved for indexing package resorces only.
- [x] Component classnames refactored.
- [x] isFitted.
- [x] Custom-types demo article added.
- [x] Remove Secondary
Secondary is sui-theme deprecated variable. The agreement is to use accent. In case of differentiating between both cases (secondary != accent) plz extend types in your brand like custom-types article example.

**Review by andresin87 — APPROVED** [review]

**Pull request merged**